kintsugi/geth/post-merge.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -x
+
+# Parse args to get elPath, ttd, dataDir
+. $(dirname "$0")/../parse-args.sh
+
+env TTD=$ttd envsubst < $currentDir/genesisPost.tmpl > $dataDir/genesis.json
+$elPath/geth --catalyst --datadir $dataDir init $dataDir/genesis.json
```

Comment:
Make ENV variables UPPER_CASE to understand faster what are the required parameters for this script